Source/WebCore/rendering/RenderBlock.cpp

```diff
@@ -334,19 +334,26 @@ void RenderBlock::styleWillChange(StyleDifference diff, const RenderStyle& newSt
     RenderBox::styleWillChange(diff, newStyle);
 }
 
-static bool borderOrPaddingLogicalWidthChanged(const RenderStyle& oldStyle, const RenderStyle& newStyle)
+bool RenderBlock::scrollbarWidthDidChange(const RenderStyle& oldStyle, const RenderStyle& newStyle)
+{
+    return (includeVerticalScrollbarSize() || includeHorizontalScrollbarSize()) && oldStyle.scrollbarWidth() != newStyle.scrollbarWidth();
+}
+
+bool RenderBlock::borderOrPaddingLogicalWidthChanged(const RenderStyle& oldStyle, const RenderStyle& newStyle)
```

Comment:
This function no longer does what it says on the box; now it's about whether the contentBox size has changed (see `RenderBox::contentBoxRect()` so it should probably be renamed.

Reply:
Fixed.